cordova-lib/src/cordova/lazy_load.js

```diff
@@ -284,4 +286,47 @@ function custom(platforms, platform) {
     });
 }
 
+// Returns a promise
+function git_clone(repository) {
```

Comment:
cordova-lib already has code to checkout from git:
https://github.com/apache/cordova-lib/blob/master/cordova-lib/src/plugman/util/plugins.js#L49

Would be better to make it so that both paths use the same helper for checking out a git repo.